vignettes/Data_retrieval.Rmd

```diff
@@ -649,3 +651,146 @@ tcplPlot(lvl = 5, # data level
          output = "pdf", # output as pdf
          fileprefix = "output/upitt") # prefix of the filename
 ```
+
+## Evaluate ToxCast AEDs for a single chemical and target
+
+This section will explore how one can compare in vivo Points of Departure (PODs) from the Toxicity Reference Database (ToxRefDB, https://www.epa.gov/comptox-tools/downloadable-computational-toxicology-data#AT) with administered equivalent doses (AEDs) from ToxCast in vitro bioactivity data (invitrodb, https://www.epa.gov/comptox-tools/exploring-toxcast-data). The process can be adapted for any given chemical and target depending on available data in either database.
+
+The following example will consider "Pentachlorophenol" and "liver toxicity"
+
+### Consider *in vivo* toxicity benchmarks as POD-Traditional
+
+First, export ToxRefDB batch download results for any chemical from the CompTox Chemicals Dashboard (https://comptox.epa.gov/dashboard/batch-search) or Hazard APIs (https://api-ccte.epa.gov/docs/) 
+
+After loading all chemical-specific data for "Pentachlorophenol", filter results to only include "liver"-related effects
+```{r txrf-data-liver-subset, eval=FALSE}
+toxref_chnm_liver <- toxref_batch_download_chnm %>% 
+  filter(endpoint_target == 'liver') 
+```
+
+Next identify the observed lowest effect (based on statistically significant i.e. treatment related=1) and lowest observed adverse (i.e. critical) effect levels at minimum dose_adjusted (mg/kg/day) value
+
+```{r txrf-lel-determination, eval=FALSE}
+toxref_chnm_liver_lel<-toxref_chnm_liver %>% 
+  summarise(lel=min(dose_adjusted[treatment_related==1]), 
+            loael=min(dose_adjusted[critical_effect==1]))
+```
+
+### Consider ToxCast *in vitro* bioactivity data as POD-NAM
+
+First, query the invitrodb database for all assay annotations, and filter results to consider only "liver" tissue-based endpoints.
+
+```{r txct-annotations-liver-subset, warning=FALSE, eval=FALSE}
+toxcast_annotations_subset <- tcplQuery("SELECT * FROM assay
```

Comment:
`tissue_annotations_subset <- tcplLoadAeid(fld = "tissue", val = "liver", add.fld = "tissue")`

Reply:
should try to avoid tcplQuery so users don't have to know db schema

Reply:
That's so much easier. Didn't realize I could query annotation fields like that